scalyr_agent/agent_main.py

```diff
@@ -865,14 +865,17 @@ def __run(self, controller):
                     logs_initial_positions = None
 
                 # 2->TODO it was very helpful to see what python version does agent run on. Maybe we can keep it?
+
+                python_version_str = ".".join(map(six.text_type, sys.version_info[:3]))
```

Comment:
I'm fine with only including the version part (aka the change you made), since it makes parsing, etc. easier, but it also doesn't hurt to include the full version string (current behavior).

Reply:
Agree.